<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlarmWindowClass</class>
 <widget class="QDialog" name="FlarmWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flarm Radar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="mapOrientationLabel">
           <property name="text">
            <string>&amp;Map orientation:</string>
           </property>
           <property name="buddy">
            <cstring>mapOrientationInput</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mapOrientationInput">
           <property name="suffix">
            <string>Â°</string>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="northButton">
           <property name="text">
            <string>&amp;North</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reverseButton">
           <property name="text">
            <string>&amp;Reverse</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="FlarmMapWidget" name="flarmMap"/>
       </item>
      </layout>
     </widget>
     <widget class="SkTableView" name="flarmTable">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkTableView</class>
   <extends>QTableView</extends>
   <header>src/gui/widgets/SkTableView.h</header>
  </customwidget>
  <customwidget>
   <class>FlarmMapWidget</class>
   <extends>QFrame</extends>
   <header>src/flarm/FlarmMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FlarmWindowClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
